### PR TITLE
Add microphone realtime example

### PR DESCRIPTION
**Pull request by HennerM**
Quick example how microphone input can be read and transcribed with the `sound device` and speechmatics libraries

### CODE REVIEW

File: examples/transcribe_from_microphone.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+
+async def transcribe_from_device(device, speechmatics_client, language: str):
+    frame_rate = 44_100
+    with sd.RawInputStream(
+        device=device, channels=1, samplerate=frame_rate, dtype="float32"
+    ) as stream:
+        settings = speechmatics.models.AudioSettings(
+            sample_rate=frame_rate,
+            chunk_size=1024 * 4,
```
Comment (aaronng91) [inline examples/transcribe_from_microphone.py:26]: where does this chunk size value come from? maybe add  a comment?

File: examples/transcribe_from_microphone.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+            encoding="pcm_f32" + ("le" if sys.byteorder == "little" else "be"),
+        )
+        # Define transcription parameters
+        conf = speechmatics.models.TranscriptionConfig(
+            language="en",
+            operating_point="enhanced",
+            max_delay=2,
```
Comment (aaronng91) [inline examples/transcribe_from_microphone.py:33]: maybe this can be a user input? this can vary the performance of RT depending on the delay

**Review by aaronng91 — APPROVED** [review]
thanks for adding such an example!

File: examples/transcribe_from_microphone.py (hunk 2 above)
Comment (HennerM) [reply examples/transcribe_from_microphone.py]: yes good idea, changed!

File: examples/transcribe_from_microphone.py (hunk 1 above)
Comment (HennerM) [reply examples/transcribe_from_microphone.py]: I removed it now, it was set to the default value anyway

**Pull request merged**